addons/mail/static/src/views/fields/properties_field/property_value.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { PropertyValue } from "@web/views/fields/properties/property_value";
+import { TagsList } from "@web/core/tags_list/tags_list";
+import { useOpenChat } from "@mail/web/open_chat_hook";
+
+// Many2one
+patch(PropertyValue.prototype, "mail.PropertyValue", {
+    setup() {
+        this._super();
+
+        if (this.env.services["mail.thread"]) {
+            // work only for the res.users model
+            this.openChat = useOpenChat("res.users");
+        }
+    },
+
+    _onAvatarClicked() {
+        if (this.openChat && this.showAvatar && this.props.comodel === "res.users") {
```

Comment:
technically, showAvatar seems more a display option, why not integrating it in openChat computation instead ?